public_html/wp-content/plugins/camptix/addons/field-country.php

```diff
@@ -38,6 +38,10 @@ function question_field_country( $name, $user_value, $question, $required = fals
 			name="<?php echo esc_attr( $name ); ?>"
 			<?php if ( $required ) echo 'required'; ?>
 		>
+			<option value="" disabled <?php selected( '', $user_value ); ?>>
```

Comment:
This works in Chrome, but Firefox doesn't default to this option, I think because it's `disabled`:
| Chrome | Firefox |
|------|------|
| <img width="763" alt="Screen Shot 2022-03-30 at 4 56 53 PM" src="https://user-images.githubusercontent.com/541093/160929517-27b6d4a4-8fe9-456a-aa48-9358164504af.png"> | <img width="750" alt="Screen Shot 2022-03-30 at 4 55 39 PM" src="https://user-images.githubusercontent.com/541093/160929314-ca9f4b05-220c-47b6-b00e-e2e84766bfc1.png"> |

Reply:
Hmm, I tested with Firefox Developer Edition 115.0b6, Firefox 115.0.2, Safari 16.4.1 and Chrome 114.0.5735.198, and all defaulted to select. Can you test again, in FF has changed something?